Products/CMFPlone/browser/templates/main_template.pt

```diff
@@ -23,7 +23,7 @@
     <metal:cache tal:replace="structure provider:plone.httpheaders" />
 
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta charset="UTF-8" />
```

Comment:
I prefer having ``UTF-8`` written in lower case. It doesn't matter technically, but it's a matter of style.